docs/helm/configure-namespaced-helm-repos.md

```diff
@@ -0,0 +1,85 @@
+# Configure Namespace Scoped Chart Repositories using namespaced CRDs
+
+While the cluster-scoped CRD `HelmChartRepository` for Helm repository provides ability for admins to add Helm repositories as custom resources, regular users/developers need an admin's permission in order to add Helm repositories of their choice.
+
+To avoid permission elevation, use namespace scoped CRD `ProjectHelmChartRepository`.
+
+## Basic structure of `ProjectHelmChartRepository`
+
+```yaml
+apiVersion: helm.openshift.io/v1beta1
+kind: ProjectHelmChartRepository
+metadata:
+  name: my-enterprise-chart-repo
+spec:
+  url: https://my.chart-repo.org/stable
+
+  # optional name that might be used by console
+  name: myChartRepo
+
+  # optional and only needed for UI purposes
+  description: my private chart repo
+
+  # required: chart repository URL
+  connectionConfig:
+    url: HELM_CHART_REPO_URL
+```
+
+## Getting Started with `ProjectHelmChartRepository`
+
+To add a new namespace-scoped Helm repository `stable` to a desired namespace `my-namespace`:
+
+```bash
+$ cat <<EOF | oc apply --namespace my-namespace -f -
+apiVersion: helm.openshift.io/v1beta1
+kind: ProjectHelmChartRepository
+metadata:
+  name: stable
+spec:
+  url: https://kubernetes-charts-incubator.storage.googleapis.com
+  displayName: Public Helm stable charts
+  description: Public Helm stable charts hosted on HelmHub
+EOF
+
+$ kubectl get projecthelmchartrepositories --namespace my-namespace
+NAME          AGE
+stable        1m
+```
+
+Note that the addition of namespace-scoped Helm repository does not impact behavior of the existing cluster-scoped Helm repository:
```

Comment:
Suggestion: These examples and output are implicit. We can avoid the repetition of the same examples. It will be good to call out the differences between `ProjectHelmChartRepository` and `HelmChartRepository` (_if any_ that we have _not_ covered yet) explicitly in the form of simple sentences.